src/composable_kernel/composable_kernel/include/utility/config.hpp

```diff
@@ -14,8 +14,9 @@
 // should enable one and only one GPU target
 #if !(defined(CK_AMD_GPU_GFX803) || defined(CK_AMD_GPU_GFX900) || defined(CK_AMD_GPU_GFX906) ||    \
       defined(CK_AMD_GPU_GFX940) || defined(CK_AMD_GPU_GFX908) || defined(CK_AMD_GPU_GFX90A) ||    \
-      defined(CK_AMD_GPU_GFX1030) || defined(CK_AMD_GPU_GFX1031) || defined(CK_AMD_GPU_GFX1100) || \
-      defined(CK_AMD_GPU_GFX1101) || defined(CK_AMD_GPU_GFX1102))
+      defined(CK_AMD_GPU_GFX941) || defined(CK_AMD_GPU_GFX942) || defined(CK_AMD_GPU_GFX1030) ||   \
```

Comment:
[Recommendation] @muralinr In similar cases, please add new items to the end -- to make diffs smaller and reviewer's job easier.

Reply:
Sure, will add going forward.

Reply:
@JehandadKhan Yeah, until we maintain possibility to enable deprecated kernels for the new targets.